openfisca_core/simulations.py

```diff
@@ -132,12 +132,13 @@ def clone(self, debug = False, debug_all = False, trace = False):
             if key not in ('debug', 'debug_all', 'trace', 'entities'):
```

Comment:
>What about the traceback ? Add it to the keys to ignore ?

It was not ignored before this PR, nor before `14.1.0`. As we don't fully understand the subtlety of this code, I'd prefer to try adding things without exactly knowing why.